tools/devops/automation/templates/build/api-diff-stage.yml

```diff
@@ -35,22 +35,22 @@ parameters:
   type: string
 
 jobs:
-- job: configure
-  displayName: 'Configure build'
-  pool:
-    vmImage: windows-latest
-
-  variables:
-    isMain: $[eq(variables['Build.SourceBranch'], 'refs/heads/main')]
-    isScheduled: $[eq(variables['Build.Reason'], 'Schedule')]
-    BRANCH_NAME: $[ replace(variables['Build.SourceBranch'], 'refs/heads/', '') ]
-
-  steps:
-  - template: ../common/configure.yml
-    parameters: 
-      repositoryAlias: ${{ parameters.repositoryAlias }}
-      commit: ${{ parameters.commit }}
-      uploadArtifacts: false
+#- job: configure
+#  displayName: 'Configure build'
+#  pool:
+#    vmImage: windows-latest
+
+#  variables:
+#    isMain: $[eq(variables['Build.SourceBranch'], 'refs/heads/main')]
+#    isScheduled: $[eq(variables['Build.Reason'], 'Schedule')]
+#    BRANCH_NAME: $[ replace(variables['Build.SourceBranch'], 'refs/heads/', '') ]
+
+#  steps:
+#  - template: ../common/configure.yml
+#    parameters: 
+#      repositoryAlias: ${{ parameters.repositoryAlias }}
+#      commit: ${{ parameters.commit }}
+#      uploadArtifacts: false
```

Comment:
Can all the commented out code be removed?

Reply:
Yes.